cis194/week2/stevemao/LogAnalysis.hs

```diff
@@ -0,0 +1,13 @@
+{-# OPTIONS_GHC -Wall #-}
+module LogAnalysis where
+import Log
+
+parseMessage :: String -> LogMessage
+parseMessage a = case head . words $ a of
```

Comment:
`head`, `read` and `!!` are not safe.(they are partial functions)
We can try to use the safe versions, but that will lead us to use `>>=`, which we have not covered yet.
So a better way for now is to pattern match on `a` itself. Remember that `’a’`, `’b’`, `’c’`... can be thought of constructors for type Char. And String is just `[Char]`. 
With a little help of guard on top of pattern match, it should be possible to avoid partial functions.

Reply:
@shineli1984 I'm still not sure how to convert `String` to `Int` here